packages/eslint-plugin/configs/custom.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
 		'@wordpress/no-unused-vars-before-return': 'error',
 		'@wordpress/valid-sprintf': 'error',
 		'@wordpress/no-base-control-with-label-without-id': 'error',
+		'@wordpress/avoid-unguarded-getrangeat': 'error',
```

Comment:
I wonder if it should be written as `get-range-at`, rather than `getrangeat`, since the camelCase casing would imply that it's in-fact three words in its expanded form ("get range at").

Lodash, for example, would convert this with its `kebabCase` function:

```
_.kebabCase( 'getRangeAt' );
// 'get-range-at'
```